steps/make-recall-decisions/start-recommendation.ts

```diff
@@ -1,12 +1,47 @@
 import { expect, type Page } from '@playwright/test'
+import { recallDateFormatter } from '../delius/utils/date-time.js'
 
-export const startRecommendation = async (page: Page, crn: string, name: string) => {
+export const searchForPersonToRecommend= async (page: Page, crn: string, name: string) => {
     await page.getByRole('button', { name: 'Start now' }).click()
     await searchForPerson(page, crn, name)
+}
+
+export const startRecommendation = async (page: Page) => {
     await page.getByRole('link', { name: 'Make a recommendation' }).click()
     await page.getByRole('button', { name: 'Continue' }).click()
 }
 
+export const verifyRecallOffenderDetails = async ( page: Page, crnInDelius: string, dobInDelius: Date, genderInDelius: string): Promise<void> => {
+    const crnInRecall = page.locator('[data-qa="personalDetailsOverview-crn"]')
+    const dobInRecall = page.locator('[data-qa="personalDetailsOverview-dateOfBirth"]')
+    const genderInRecall = page.locator('[data-qa="personalDetailsOverview-gender"]')
+    await expect(crnInRecall).toContainText(crnInDelius)
+    await expect(dobInRecall).toContainText(recallDateFormatter(dobInDelius))
```

Comment:
Given this is only used in one place, it might be worth just inlining it for now?
```suggestion
    await expect(dobInRecall).toContainText(dobInDelius.toLocaleDateString('en-GB', { day: 'numeric', month: 'long', year: 'numeric' }))
```